app/models/payment.rb

```diff
@@ -161,13 +161,19 @@ def card
     end
 
     def status
-      if @bt_result.success?
+      if successful?
         Payment::BraintreeTransaction.statuses[:success]
       else
         Payment::BraintreeTransaction.statuses[:failure]
       end
     end
 
+    def successful?
+      return @bt_result.success? if @bt_result.respond_to?(:success?)
+      return true if @bt_result.class == Braintree::WebhookNotification::Kind::SubscriptionChargedSuccessfully
+      false
+    end
+
```

Comment:
but that doesn't do what I want it to do. if it responds to `success?`, I want to return it's value of `success?`, even if that's false.